providers/aws/resources/aws_timestream.go

```diff
@@ -0,0 +1,208 @@
+// Copyright (c) Mondoo, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package resources
+
+import (
+	"context"
+	"github.com/aws/aws-sdk-go-v2/service/timestreamwrite"
+	"github.com/rs/zerolog/log"
+	"go.mondoo.com/cnquery/v11/llx"
+	"go.mondoo.com/cnquery/v11/providers-sdk/v1/util/convert"
+	"go.mondoo.com/cnquery/v11/providers-sdk/v1/util/jobpool"
+	"go.mondoo.com/cnquery/v11/providers/aws/connection"
+	"golang.org/x/exp/slices"
+)
+
+// AWS TimeStream LifeAnalytics is not available in all regions
+var timeStreamLifeRegions = []string{
+	"us-gov-west-1",
+	"ap-south-1",
+	"ap-northeast-1",
+	// "ap-southeast-1", // only InfluxDB is available
+	"ap-southeast-2",
+	"eu-central-1",
+	"eu-west-1",
+	// "eu-north-1", // only InfluxDB is available
+	"us-east-1",
+	"us-east-2",
+	"us-west-2",
+}
+
+func (a *mqlAwsTimestreamLifeanalytics) id() (string, error) {
+	return "aws.timestream.lifeanalytics", nil
+}
+
+func (a *mqlAwsTimestreamLifeanalytics) databases() ([]interface{}, error) {
+	conn := a.MqlRuntime.Connection.(*connection.AwsConnection)
+	res := []interface{}{}
+	poolOfJobs := jobpool.CreatePool(a.getDatabases(conn), 5)
+	poolOfJobs.Run()
+
+	// check for errors
+	if poolOfJobs.HasErrors() {
+		return nil, poolOfJobs.GetErrors()
+	}
+	// get all the results
+	for i := range poolOfJobs.Jobs {
+		if poolOfJobs.Jobs[i].Result != nil {
+			res = append(res, poolOfJobs.Jobs[i].Result.([]interface{})...)
+		}
+	}
+
+	return res, nil
+}
+
+func (a *mqlAwsTimestreamLifeanalytics) getDatabases(conn *connection.AwsConnection) []*jobpool.Job {
+	tasks := make([]*jobpool.Job, 0)
+	regions, err := conn.Regions()
+	if err != nil {
+		return []*jobpool.Job{{Err: err}}
+	}
+
+	for _, region := range regions {
+		regionVal := region
+		if !slices.Contains(timeStreamLifeRegions, regionVal) {
+			continue
```

Comment:
nit: can we log a debug msg here that we're skipping since its unsupported? this can help us down the line if we're debugging and wondering why things arent coming back

Reply:
Added new debug message